quolance-api/src/main/java/com/quolance/quolance_api/services/impl/PortfolioServiceImpl.java

```diff
@@ -0,0 +1,68 @@
+package com.quolance.quolance_api.services.impl;
+
+import com.quolance.quolance_api.dtos.PortfolioDto;
+import com.quolance.quolance_api.entities.Portfolio;
+import com.quolance.quolance_api.entities.User;
+import com.quolance.quolance_api.repositories.PortfolioRepository;
+import com.quolance.quolance_api.repositories.UserRepository;
+import com.quolance.quolance_api.services.PortfolioService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+@Service
+public class PortfolioServiceImpl implements PortfolioService {
+
+    
+    private final PortfolioRepository portfolioRepository;
+    private final UserRepository userRepository;
+
+    @Autowired
+    public PortfolioServiceImpl(PortfolioRepository portfolioRepository, UserRepository userRepository) {
```

Comment:
Remove the constructor, just add @ RequiredArgsConstructor on the class to have it made automatically